### PR TITLE
Removed units from the <img> tag

### LINKED ISSUE #4549: Incorrect (but compatible) width/height attributes in image markup
## Description

Many of the `` tags in SecureDrop's templates look like this:

```html
<img class="icon" src="some-image.png" width="20px" height="14px">
```

To the CSS-trained eye, this may look perfectly correct, and indeed it is rendered without problems as most rendering engines will happily ignore anything after the integer. However, this is technically not valid HTML, and https://validator.w3.org/ will report it as an error ("Bad value 14px for attribute height on element img: Expected a digit but saw p instead."). 

For `width` and `height` attributes, the value should be a non-negative integer without a unit (as pixels are always the unit). To avoid surprises, let's use valid markup throughout.

### PR DESCRIPTION
**Pull request by nightwarrior-xxx**
## Status

Ready for review

## Description of Changes

Fixes #4549 

Changes proposed in this pull request:

## Testing

How should the reviewer test this PR?

As suggested by @eloquence I have remove units from the image tag.

### CODE REVIEW

**Comment by nightwarrior-xxx** [comment]
@kushaldas Please take a look when you are free.

File: securedrop/journalist_templates/config.html [hunk 1]
```diff
@@ -33,7 +33,7 @@ <h2>{{ gettext('Logo Image') }}</h2>
     <br>
   </p>
   <h5>
-    {{ gettext('Recommended size: 500px * 450px') }}
+    {{ gettext('Recommended size: 500 * 450') }}
```
Comment (eloquence) [inline securedrop/journalist_templates/config.html]: no reason to remove units here

**Comment by nightwarrior-xxx** [comment]
@eloquence Lint is failing again :(

**Comment by eloquence** [comment]
> Lint is failing again :(

Can you rebase your PR against the current state of `develop` in the main repo to see if that fixes things?

**Comment by nightwarrior-xxx** [comment]
@eloquence Merging conflicts when I tried that. looks like I have to clone the whole repo again .

**Pull request closed without merging**